### ISSUE TITLE
Burndown charts should ignore pull requests

### ISSUE THREAD

**Issue by GordonSmith** [ZenHubIO/support#766]
We assign milestones to our pull requests to help us prioritize our code reviews.  
But once we do that they appear on the burndown page (highlighted in red):
![image](https://cloud.githubusercontent.com/assets/966863/15740932/1afb9580-28af-11e6-8921-71b29fde0322.png)

**Comment by GordonSmith** [comment]
For bonus points adjust the burndown chart to include the PRs (IOW extend the burndown line with a dotted extension showing where it would be if all the PRs were pulled).

**Comment by azenMatt** [comment]
Hi @GordonSmith 

This is now possible - on the latest version (2.23.15) you can toggle between showing and not showing PR's 

![image](https://cloud.githubusercontent.com/assets/7584089/17605721/aeb569ce-5fd0-11e6-84a2-623e4b3f778d.png)


![image](https://cloud.githubusercontent.com/assets/7584089/17605739/c309cc3a-5fd0-11e6-8679-dd0d6da387dd.png)

Does this fit what you need?

**Comment by GordonSmith** [comment]
@azenMatt Yes, that addresses the original bug report, thx.  The other suggestion above I believe would be really nice, is to extend the chart with a dotted line showing where it "will" be once any outstanding PRs are pulled.  

It might only be me, but we assign our PRs a milestone so that we can prioritize which ones need to be reviewed first.  Also our PRs tend to be around a long time, so seeing that we "would" be on schedule "if" they were pulled would be a nice addition.

**Comment by azenMatt** [comment]
Thanks @GordonSmith - lets leave this issue open to track that second request
